manifests/exception.pp

```diff
@@ -53,10 +53,10 @@
 #   windows_firewall::exception { 'WINRM-HTTP-In-TCP':
 #     ensure       => present,
 #     direction    => 'in',
-#     action       => 'Allow',
-#     enabled      => 'yes',
```

Comment:
I'm all for the change from 'yes'/'no' to true/false, though since this gets rid of the old functionality, it will need to be a major version bump. Fine with me, though extra attention should be paid to the module to see if there are other areas that should change, so we are not quickly iterating through major releases.

Reply:
@ghoneycutt, do you see anything that should be changed together with this? I could invest a bit more time if necessary.

Reply:
Looked through the module and I don't see anything that is begging to be changed and needs a major version bump. The `init.pp` should be updated with the rest. Would be useful to go through the open issues and see if those can be closed or addressed as those might bring up changes that would need a major version bump. I saw another PR to transition to types and providers, though I don't think that needs to be included with what you've got going on here.

Reply:
I looked at the other PR, but I don't see any conflicts or reasons to put them together.
Regarding the other issues. I went through them and commented where I had some input. Some of them can be simply closed.
I'm on your side that ```init.pp``` can be updated as well (data types wise), but I'd prefer to do this in some separate step in order to get this PR done :smile: